<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>335</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Tonique</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QFrame" name="frame">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="1">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Nombre de mesures</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QSpinBox" name="tempo">
         <property name="minimum">
          <number>20</number>
         </property>
         <property name="maximum">
          <number>300</number>
         </property>
         <property name="value">
          <number>60</number>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QComboBox" name="note">
         <item>
          <property name="text">
           <string>Do</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>RÃ©</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Mi</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Fa</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Sol</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>La</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Si</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string/>
          </property>
         </item>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Tempo</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Note</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QSpinBox" name="nbMesures">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="value">
          <number>4</number>
         </property>
        </widget>
       </item>
       <item row="1" column="3">
        <widget class="QFrame" name="frame_2">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLCDNumber" name="un">
            <property name="segmentStyle">
             <enum>QLCDNumber::Flat</enum>
            </property>
            <property name="value" stdset="0">
             <double>1.000000000000000</double>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLCDNumber" name="deux">
            <property name="segmentStyle">
             <enum>QLCDNumber::Flat</enum>
            </property>
            <property name="value" stdset="0">
             <double>2.000000000000000</double>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLCDNumber" name="trois">
            <property name="segmentStyle">
             <enum>QLCDNumber::Flat</enum>
            </property>
            <property name="value" stdset="0">
             <double>3.000000000000000</double>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLCDNumber" name="quatre">
            <property name="segmentStyle">
             <enum>QLCDNumber::Flat</enum>
            </property>
            <property name="value" stdset="0">
             <double>4.000000000000000</double>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QFrame" name="frame_4">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Mesure:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLCDNumber" name="mesure"/>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QFrame" name="frameDessin">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>1</verstretch>
       </sizepolicy>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2"/>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QFrame" name="frame_3">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="uneFois">
         <property name="text">
          <string>1 fois</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="startStop">
         <property name="text">
          <string>Start</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <tabstops>
  <tabstop>note</tabstop>
  <tabstop>nbMesures</tabstop>
  <tabstop>tempo</tabstop>
  <tabstop>uneFois</tabstop>
  <tabstop>startStop</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
